mats_metadata_and_indexes/index_creation_scripts/README.md

```diff
@@ -1,11 +1,21 @@
-# How to build the indexes
+# Indexes
+
+## Overview
+
+### Cluster roles
+
+#### ingest / development
+The standalone cluster on adb-cb1 is used for ingest data creation and import. The ingest processes query this cluster and the import processes import to this cluster.
+#### production
+The production cluster is what the apps read. This cluster is populated by xdcr from the ingest cluster
 
 ## Index Creation Scripts
+We are trying to keep an up to date N1QL script for the standalone cluster and a different one for the production cluster (they are not the same index set) that contain the index creation statements necessary to recreate a proper set of indexes for either of those environments, or for a new environment that has one of thse roles. 
 
 There are three index creation scripts
 
 - mats_metadata_and_indexes/index_creation_scripts/create_indexes-0-replicas.n1ql
-  This is for the standalone server since it cannot support index replication (only one node)
+  This is for the standalone server since it cannot support index replication (it only has one node)
   Copy the contents of this N1QL script and either execute it in the query window of the UI of the
   standalone server or execute it with the command line interface cbq.
   After that you must also execute the N1QL in the build_indexes.n1ql script.
```

Comment:
I removed it because it was all wrong. I am hoping to use the backup to create a new kind of index creation script, maybe not named the same but created from the backup. I have an issue and a branch open for that work but it isn't done. Perhaps we should make these changes there when that work is finished. I probably should not have removed the scripts on this branch.

Reply:
im ok either way.

Reply:
Okay. They definitely should be updated but I'm okay with doing it in either branch. If all else is equal, I like coupling documentation updates in the same branch as the code changes so they don't get forgotten.

If you wanted to get really fancy, you could also remove these changes from this branch/PR as well so they'd all happen in the right feature branch. That'd be some moderate git surgery though.